src/ctelnet.cpp

```diff
@@ -441,10 +445,8 @@ void cTelnet::handle_socket_signal_connected()
     QString nothing = "";
     mpHost->mLuaInterpreter.call(func, nothing);
     mConnectionTime.start();
-    if ((mpHost->getLogin().size() > 0) && (mpHost->getPass().size() > 0)) {
-        mTimerLogin->start(2000);
-        mTimerPass->start(3000);
-    }
+    mTimerLogin->start(2000);
```

Comment:
:+1:  for fixing things so that it still sends the user name even if there isn't a password stored...

Reply:
Yep!

But I realised I introduced a bug, if there was no username and just a password, it'd just send that... that seems wrong, all games require at leas a character name or an account name. Fixed.